src/data/_speakers.yaml

```diff
@@ -3,169 +3,180 @@
 ####################################################################
 id_liaoruiqi:
   name: 廖瑞奇
- 
+  name_en: "Ruiqi Liao"
   city_shanghai: 1
   avatar: ./asset/images/avatar-2017/speakers/liaoruiqi.jpg
   avatar_en: ../asset/images/avatar-2017/speakers/liaoruiqi.jpg
-  intro: "达达 - 京东到家配送算法团队负责人,  在机器学习、物流算法、计算广告等领域有着丰富的实践经验。2016 年初加入达达，从 0 到 1 建立起达达的算法团队。目前主要负责订单指派、动态定价、路径规划等相关工作，希望利用人工智能的力量改变同城配送的效率与成本，实现更为智能的即时物流。"
-  intro_en: ""
+  intro: "达达 - 京东到家配送算法团队负责人，在机器学习、物流算法、计算广告等领域有着丰富的实践经验。2016 年初加入达达，从 0 到 1 建立起达达的算法团队。目前主要负责订单指派、动态定价、路径规划等相关工作，希望利用人工智能的力量改变同城配送的效率与成本，实现更为智能的即时物流。"
+  intro_en: "Leader of Dada-Jingdong Home distribution algorithm team, Ruiqi Liao have rich experience in machine learning, logistics algorithm and computing advertising. In early 2016, Liao joined Dada, and build the algorithm team from scratch. Now, he is mainly responsible for order assignment, dynamic pricing, path planning and some related work, wishing to optimize the effectiveness and cost of distribution with the power of AI, and achieve more intelligent real-time logistics."
```

Comment:
Ruiqi Liao have rich experience -> has
build -> built
wishing to optimize the effectiveness and cost of distribution with the power of AI -> reduce cost